PxtlCa.XmlCommentMarkDownGenerator/XmlToMarkDown.cs

```diff
@@ -84,8 +84,7 @@ public static string ToMarkDown(this XNode node, string assemblyName = null)
 
 
             if (node.NodeType == XmlNodeType.Text)
-                return Regex.Replace(((XText)node).Value.Replace('\n', ' '), @"\s+", " ");
-
+                return Regex.Replace(((XText)node).Value.Replace(@"            ", ""), @"^\n|[S*|\n]$","",RegexOptions.Multiline);
```

Comment:
Crud.  I see your PR is based on the other PR.  We'll have to resolve that first.